### DISCUSSION TITLE
Unable to Clone GitHub Repository in Linux Terminal

### DISCUSSION THREAD

**Discussion by ghost** [learndevopsonline/learndevopsonline#1478] [Q&A]
Hi @r-devops ,

I am Unable to clone GitHub Repository in Linux Terminal.
It is showing fatal error.

Could you please help me?
![Screenshot 2023-09-22 211840](https://github.com/learndevopsonline/learndevopsonline/assets/145763311/e10993df-f547-4162-ab9f-3f873c983101)

**Comment by sreedharm07** [comment]


**Reply by ghost** [reply]
ok I will try

**Reply by ghost** [reply]
I have created repository name **learning-ansible1**

**Reply by sreedharm07** [reply]
sorry its my mistake i dint seen name properly. 
i have cloned your repo in my machine its cloned properly. 
can you try now once.

**Reply by ghost** [reply]
ok ,Thank you bro
